WordPressAuthenticator/Unified Auth/GoogleAuthenticator.swift

```diff
@@ -47,6 +84,7 @@ class GoogleAuthenticator: NSObject {
     private override init() {}
     var loginDelegate: GoogleAuthenticatorLoginDelegate?
     var signupDelegate: GoogleAuthenticatorSignupDelegate?
+    var delegate: GoogleAuthenticatorDelegate?
```

Comment:
This property should be weak (and the related protocol, class bound), otherwise it looks like an instance of `GoogleAuthViewController` is retained after the login completes. Here's a screenshot of the memory graph showing that.

<img width="247" alt="Screen Shot 2020-06-03 at 12 04 41 PM" src="https://user-images.githubusercontent.com/34376330/83666651-e12f4080-a592-11ea-869e-992e54b06177.png">

Note that when I tweaked that var to weak, this did not happen. It maybe worth looking in the two existing delegates as well